recipe/build.sh

```diff
@@ -2,22 +2,43 @@
 
 set -ex
 
-if [[ "$target_platform" == linux* ]]; then
-    # do not build with MKL support
-    export TF_NEED_MKL=0
-    export BAZEL_MKL_OPT=""
-
-    mkdir -p ./bazel_output_base
-    export BAZEL_OPTS="--batch "
-
-    # Linux
-    # the following arguments are useful for debugging
-    #    --logging=6
-    #    --subcommands
-
-    # Set compiler and linker flags as bazel does not account for CFLAGS,
-    # CXXFLAGS and LDFLAGS.
-    BUILD_OPTS="
+if [[ "$target_platform" == "osx-64" ]]; then
+  export CC=clang
+  export CXX=clang++
+fi
+
+export PATH="$PWD:$PATH"
+export CC=$(basename $CC)
+export CXX=$(basename $CXX)
+export LIBDIR=$PREFIX/lib
+export INCLUDEDIR=$PREFIX/include
+
+echo "#!/bin/bash"                                > compiler-wrapper
+echo "export C_INCLUDE_PATH=$PREFIX/include"     >> compiler-wrapper
+echo "export CPLUS_INCLUDE_PATH=$PREFIX/include" >> compiler-wrapper
+chmod +x "compiler-wrapper"
+cp compiler-wrapper $CC
+cp compiler-wrapper $CXX
+echo "exec $BUILD_PREFIX/bin/$CC -L$PREFIX/lib \"\$@\""   >> $CC
```

Comment:
@jjhelmus, this is how `-L$PREFIX/lib` is sent. This is a hack though. I tried passing `LIBRARY_PATH`, but that works only for gcc configured for native compilation.